composer.json

```diff
@@ -49,7 +49,7 @@
         "symfony-cmf/resource-bundle": "^1.0",
         "symfony-cmf/testing": "^2.0.1",
```

Comment:
You probably don't want to change this version, but you will need to release 2.1 to have some change to pass the dev-master build

Reply:
correct :-(
and there the chain begins: 
in testing component i wait for a release of phpcr-odm 2.0 to not require a dev dependency in my component. (@dbu  any info on the phpcr release lane?)

Reply:
I tried to make testing component compatible with sf 4 on the 2.0 branch, but as I saw it was already on 2.1 I abandoned that PR.

Reply:
Indeed, i also did some changes/improvements on testing (no breaks)

Reply:
but, we can say: lets release this one, when `testing` is released and so whem `phpcr-odm` is released?